docs/docsite/rst/dev_guide/style_guide/basic_rules.rst

```diff
@@ -57,7 +57,7 @@ When documenting menus or commands, it helps to **bold** what is important.
 For menu procedures, bold the menu names, button names, and so on to help the user find them on the GUI:
 
 1. On the **File** menu, click **Open**.
-2. Type a name in the **User Name** field.
+2. Type a name in the **username** field.
```

Comment:
This isn't a file name change so please remove this change so we keep this PR focused on one thing.